.dockerignore

```diff
@@ -0,0 +1,22 @@
+.git/
+.github/
+config/*
+data/
+flow-typed/
+lib/
+node_modules/
+static/js/
+static/stylesheets/
+test/
+.doclets.yml
+.editorconfig
+.eslintignore
+.eslintrc.js
+.flowconfig
+.gitignore
+.gitmodules
+.tern-project
+.travis-yml
+LICENSE
+package-lock.json
```

Comment:
I think you'll want this copied into the image, otherwise `npm install` will get a slightly different dependency tree every time you build the image